src/app/core/models/album.interface.ts

```diff
@@ -0,0 +1,26 @@
+import { FormatType, RecordType } from './attribute.types';
+import { DocumentReference } from '@angular/fire/compat/firestore';
+
+export interface Album {
+  title: string;
+  artist: string;
+  record?: RecordType;
+  genres?: string[];
+  saved_by: string[];
+}
+
+export interface AlbumWithId extends Album {
```

Comment:
Has to be a separate interface as the id has to be required whenever it is needed